src/overlays/actors/ovl_En_Tg/z_en_tg.h

```diff
@@ -2,15 +2,35 @@
 #define Z_EN_TG_H
 
 #include "global.h"
+#include "objects/object_mu/object_mu.h"
 
 struct EnTg;
 
 typedef void (*EnTgActionFunc)(struct EnTg*, PlayState*);
 
+typedef struct EnTgHeartEffect {
+    /* 0x00 */ u8 isEnabled;
+    /* 0x01 */ u8 timer;
+    /* 0x02 */ char unk2[0x2];
```

Comment:
You can just remove this `unk2` and it will work fine; this is struct padding if you aren't familiar with it.